### PR TITLE
Small refactor of viz.regtools 

### PR DESCRIPTION
**Pull request by arokem**
For now, just created a function to handle some boilerplate

### CODE REVIEW

**Comment by stefanv** [comment]
Why ``_three_plot`` instead of ``_tile_plot`` that allows an arbitrary number of images?

File: dipy/viz/regtools.py [hunk 1]
```diff
@@ -4,6 +4,20 @@
...
+def _tile_plot(imgs, titles, **kwargs):
+    """
+    Helper function
+    """
+    # Create a new figure and plot the three images
+    fig, ax = plt.subplots(1, len(imgs))
+    for ii, a in enumerate(ax):
+        a.set_axis_off()
+        a.imshow(imgs[ii], **kwargs)
+        a.set_aspect('auto', 'box')
```
Comment (omarocegueda) [inline dipy/viz/regtools.py]: People may not want to set auto-aspect ratio (e.g. when using "plot_slices" the coronal and sagital slices look weird), maybe add it as an optional parameter?

Comment (arokem) [inline dipy/viz/regtools.py]: OK - I have changed it. It makes sense for users to just use the returned
figure object to customize this as desired.

**Comment by omarocegueda** [comment]
Thank you @arokem!, this looks great to me! does anybody else has any more comments? I think it is ready to merge =)

**Comment by Garyfallidis** [comment]
@arokem it seems that @omarocegueda is good with your PR. Can you remove the WIP flag. And then @omarocegueda can you merge it?

**Comment by arokem** [comment]
All done here - feel free to merge!

**Pull request merged**